tests/Moq.Tests.VisualBasic/IssueReports.vb

```diff
@@ -75,4 +75,40 @@ Public Class IssueReports
 
 	End Class
 
+	Public Class Issue1129
+
+		<Fact>
+		Public Sub Test()
+			Dim failClassMock = New Mock(Of IndexerInterface)()
```

Comment:
Consider renaming the `failClassMock` variable to just `classMock`/`mock`. (Mentioning "failure" will no longer make sense once the problem is fixed and there is no more failure!)